docs/ref/django-admin.txt

```diff
@@ -872,6 +872,19 @@ By default, the development server doesn't serve any static files for your site
 you want to configure Django to serve static media, read
 :doc:`/howto/static-files/index`.
 
+sendtestemail
+-----
+
+.. django-admin:: sendtestemail
+
+Sends a test email using to the recipient specified.
+
+e.g.::
+
+    django-admin sendtestemail foo@example.com
+
+.. versionadded:: 1.9
```

Comment:
This statement should be a few lines higher, see https://github.com/django/django/blob/stable/1.7.x/docs/ref/django-admin.txt#L670 for an example.